packages/features/ee/workflows/lib/reminders/reminderScheduler.ts

```diff
@@ -23,51 +23,53 @@ export const scheduleWorkflowReminders = async (
 ) => {
   if (workflows.length > 0 && !needsConfirmation) {
     workflows.forEach((workflowReference) => {
-      if (workflowReference.workflow.steps.length > 0) {
-        const workflow = workflowReference.workflow;
-        if (
-          workflow.trigger === WorkflowTriggerEvents.BEFORE_EVENT ||
-          workflow.trigger === WorkflowTriggerEvents.NEW_EVENT
-        ) {
-          workflow.steps.forEach(async (step) => {
-            if (step.action === WorkflowActions.SMS_ATTENDEE || step.action === WorkflowActions.SMS_NUMBER) {
-              const sendTo = step.action === WorkflowActions.SMS_ATTENDEE ? smsReminderNumber : step.sendTo;
-              await scheduleSMSReminder(
-                evt,
-                sendTo,
-                workflow.trigger,
-                step.action,
-                {
-                  time: workflow.time,
-                  timeUnit: workflow.timeUnit,
-                },
-                step.reminderBody || "",
-                step.id,
-                step.template
-              );
-            } else if (
-              step.action === WorkflowActions.EMAIL_ATTENDEE ||
-              step.action === WorkflowActions.EMAIL_HOST
-            ) {
-              const sendTo =
-                step.action === WorkflowActions.EMAIL_HOST ? evt.organizer.email : evt.attendees[0].email;
-              scheduleEmailReminder(
-                evt,
-                workflow.trigger,
-                step.action,
-                {
-                  time: workflow.time,
-                  timeUnit: workflow.timeUnit,
-                },
-                sendTo,
-                step.emailSubject || "",
-                step.reminderBody || "",
-                step.id,
-                step.template
-              );
-            }
-          });
-        }
+      if (workflowReference.workflow.steps.length === 0) return;
+
+      const workflow = workflowReference.workflow;
+      if (
+        workflow.trigger ===
+        (WorkflowTriggerEvents.BEFORE_EVENT ||
+          WorkflowTriggerEvents.NEW_EVENT ||
+          WorkflowTriggerEvents.RESCHEDULE_EVENT)
```

Comment:
This never returns true 
```suggestion
        workflow.trigger === WorkflowTriggerEvents.BEFORE_EVENT ||
        workflow.trigger === WorkflowTriggerEvents.NEW_EVENT ||
        workflow.trigger === WorkflowTriggerEvents.RESCHEDULE_EVENT
```

Reply:
oh yeah forgot to change here